tools/packaging/packaging.psm1

```diff
@@ -1719,7 +1719,7 @@ function CreateNugetPlatformFolder
         [string] $PlatformBinPath
```

Comment:
perhaps we should have one place where we define this in this script?

Reply:
@TravisEz13 Please re-review. Pushed changes to use a variable.